docs/compatibility.md

```diff
@@ -1,10 +1,18 @@
-# Compatibility with MMDetection 1.x
+# Compatibility of MMDetection 2.x
+
+## Mask AP Evaluation
+
+Before [PR 4898](https://github.com/open-mmlab/mmdetection/pull/4898), the mask AP of small, medium, and large instances is calculated based on the bounding box area rather than the real mask area. This leads to higher APs and APm but lower APl but will not affect the overall mask AP. 
```

Comment:
Should we also add mmdet version number?

Reply:
Sure, and this PR will be merged right before the new release.